### DISCUSSION TITLE
Allow to set a custom ping address

### DISCUSSION THREAD

**Discussion by dmishh** [InternetConnectionMonitor/icm-public#94] [Ideas and Feature requests]
Add an ability to change the ping address to

1. A different one (predefined list like google, microsoft, baidu, etc)
2. A custom one (user-defined HTTP or HTTPs endpoint)

**Comment by CodesWatching** [comment]
baidu.com

**Comment by janwilmans** [comment]
It would be nice to specify a custom ip-address or hostname so you can also use it monitor a specific connection, not just 'the internet gateway'

**Comment by oeda** [comment]
Agree with janWilmans, it would be very handy to be able to monitor the connection between two sites to get an idea on the quality of the two ISPs involved.

**Comment by EllysAlvares** [comment]
Senhores, estou com serios problemas a respeito disso. A operadora ao qual presta serviço de internet, ela todos os dias dá o resset da sua base de dados, mas o ping continua normal, mas a minha conexão cai. Isso ocorre todos os dias as 8h da manhã, horário do Brasil. Voces tem alguma idea de como pegar essa desconexão? será que mudando o ping para o servidor ao qual presto serviço, resolveria??

**Comment by ghost** [comment]
I have had an issue of being hacked. Is there a way to track or identify who is on my current wifi system. I'm not the most tech savvy person but I am doing my best to learn as much as I can as fast as possible. Thanks. Great peace and Love I wish to you all.

**Comment by iantc** [comment]
Google is not always accessible for those who must travel for work due to local restrictions. Thank you very much for considering this request.

**Comment by agusaza** [comment]
se logro agregar para poder decir a donde se quieres hacer ping de esta aplicacion?

**Comment by sineomatic** [comment]
a custom IP or domain would be a great addition. Please add!

**Comment by marc-ids-online** [comment]
Please include an option to be able to choose our own IP or URL to ping.

**Comment by JohnZhong505** [comment]
It would be very useful to customize the ping URL, please add!

**Comment by DictoDictov** [comment]
I would be a nice functionality. And even more — I recommend to make a list of popular alternative domains such as Yahoo, Cloudflare, Github etc.

**Comment by UncleJS** [comment]
Please include an option to be able to choose our own IP or URL to ping.

**Comment by dmishh** [comment]
The work has been started in #127 
Target release - 6.0.0, ETA end of February 2025

**Comment by emeyedeejay** [comment]
Great to hear work has started - how is it going? Definitely will be an excellent feature!

**Comment by dmishh** [comment]
#127 is finally done!
The new version is on review in the Chrome Store and should be out in the next couple of days
